src/ert/_c_wrappers/config/config_parser.py

```diff
@@ -26,6 +53,68 @@ def __init__(self, errors: str, config_file: Optional[str] = None) -> None:
             else f"{self.errors}"
         )
 
+    def replace(self, old_text: str, new_text: str):
+        return ConfigValidationError(
+            errors=self.errors.replace(old_text, new_text),
+            config_file=self.config_file,
+            location=self.location,
+        )
+
+    @property
+    def config_file(self):
+        return self.location.filename
+
+    @config_file.setter
+    def config_file(self, config_file):
+        self.location.filename = config_file
+
+    def get_error_messages(self):
+        return [self.errors]
+
+
+class CombinedConfigError(ConfigValidationError):
+    def __init__(
+        self,
+        errors: Optional[
+            List[Union[ConfigValidationError, "CombinedConfigError"]]
```

Comment:
There is no need for `Union` here as `CombinedConfigError` is a subtype of `ConfigValidationError`.